openssl/src/pkey_ctx.rs

```diff
@@ -920,6 +958,19 @@ mod test {
         ctx.keygen().unwrap();
     }
 
+    #[test]
+    #[cfg(not(boringssl))]
+    fn dsa_keygen() {
+        let mut ctx = PkeyCtx::new_id(Id::DSA).unwrap();
+        ctx.paramgen_init().unwrap();
+        ctx.set_dsa_paramgen_bits(2048).unwrap();
+        let params = ctx.paramgen().unwrap();
+
```

Comment:
Sorry, I was thinking more like an assert that the `size()` is correct

Reply:
Added 👍🏻